docs/data.md

```diff
@@ -32,6 +48,12 @@ In this example:
 *  `MAGENTO_ADMIN_USERNAME` is a name of an environment variable.
    The corresponding value will be assigned to `userInput` as a result.
 
+The following is the usage of `_ENV` in the `Magento/Braintree/Test/Mftf/ActionGroup/AdminDeleteRoleActionGroup.xml` action group:
```

Comment:
"The following is an example of the usage of..." 
Same with L72 and L96.

Reply:
@dobooth Updated the changes.